react/src/services/books.js

```diff
@@ -12,3 +12,33 @@ export const getBookService = (bookId) => {
     `/books/${bookId}`
   );
 };
+
+export const getBookRentsService = (bookId) => {
```

Comment:
We usually also export these methods into an object by default, like `actionCreators` are exported.

Reply:
How do you usually name that default object?